tests/pfc/files/configs/pfc.py

```diff
@@ -0,0 +1,399 @@
+                class_enable_vector=FieldPattern(choice=v),
+                pause_class_0=FieldPattern(choice=p[0]),
+                pause_class_1=FieldPattern(choice=p[1]),
+                pause_class_2=FieldPattern(choice=p[2]),
+                pause_class_3=FieldPattern(choice=p[3]),
+                pause_class_4=FieldPattern(choice=p[4]),
+                pause_class_5=FieldPattern(choice=p[5]),
+                pause_class_6=FieldPattern(choice=p[6]),
+                pause_class_7=FieldPattern(choice=p[7]),
+            ))
+
+            pause_flow = Flow(
+                name='Pause Storm',
+                tx_rx=TxRx(pause_endpoint),
+                packet=[pause],
+                size=Size(64),
+                rate=Rate('line', value=100),
+                duration=Duration(Continuous(delay=0, delay_unit='nanoseconds'))
+            )
+        elif (pause_frame_type == 'global') :
+            pause = Header(EthernetPause(
+                dst=FieldPattern(choice='01:80:C2:00:00:01'),
+                src=FieldPattern(choice='00:00:fa:ce:fa:ce')
+            ))
+
+            pause_flow = Flow(
+                name='Pause Storm',
+                tx_rx=TxRx(pause_endpoint),
+                packet=[pause],
+                size=Size(64),
+                rate=Rate('line', value=pause_line_rate),
+                duration=Duration(Continuous(delay=0, delay_unit='nanoseconds'))
+            )
+        else :
+            pass   
+
+        config.flows.append(pause_flow)
+
+    return one_hundred_gbe
+
+
+@pytest.fixture
+def start_delay(request):
+    return request
+
+
+@pytest.fixture
+def traffic_duration(request):
+    return request
+
+
+@pytest.fixture
+def bw_multiplier(request):
+    return request
+
+
+@pytest.fixture
+def pause_line_rate(request):
+    return request
+
+
+@pytest.fixture
+def traffic_line_rate(request):
+    return request
+
+
+@pytest.fixture
+def frame_size(request):
+    return request
+
+
+@pytest.fixture(scope='session')
+def serializer(request):
+    class Serializer(object):
+        def __init__(self, request):
+            self.request = request
+            self.test_name = getattr(request.node, "name")
+
+        def json(self, obj):
+            import json
+            json_str = json.dumps(obj, indent=2, default=lambda x: x.__dict__)
+            return '\n[%s] %s: %s\n' % (self.test_name, obj.__class__.__name__, json_str)
+
+        def yaml(self, obj):
+            import yaml
+            yaml_str = yaml.dump(obj, indent=2)
+            return '\n[%s] %s: %s\n' % (self.test_name, obj.__class__.__name__, yaml_str)
+
+        def obj(self, json_string):
+            return json.loads(json_string, object_hook=self._object_hook)
+
+        def _object_hook(self, converted_dict):
+            return namedtuple('X', converted_dict.keys())(*converted_dict.values())
+
+    return Serializer(request)
+
+
+@pytest.fixture
+def port_bandwidth(conn_graph_facts,
```

Comment:
Add comments

Reply:
This function is updated with appropriate comment section.